config/zones.json

```diff
@@ -2661,13 +2661,13 @@
       ]
     ],
     "capacity": {
-      "biomass": 0,
-      "coal": 4404,
-      "gas": 2115,
-      "hydro": 723,
-      "nuclear": 103,
+      "biomass": 52,
+      "coal": 0,
```

Comment:
Interesting that Delhi has no coal power station anymore ! [Supporting document](https://cea.nic.in/wp-content/uploads/pdm/2020/09/list_power_stations_2020.pdf)